calendarsubscription/calendarsubscription.go

```diff
@@ -29,6 +29,7 @@ func (cs CalendarSubscription) Normalize() (*CalendarSubscription, error) {
 	}
 
 	err := validate.Many(
+		validate.ArrayLength("ReminderMinutes", cs.Config.ReminderMinutes, 15),
```

Comment:
Rather than adding a separate function, and since we are using this at only once place as of now, could we use something like : 
`err = validate.Range("ReminderMinutes", len(cs.Config.ReminderMinutes), 0, 15)`